lib/LanguageServerCodeTransform/Model/NameImport/NameImporter.php

```diff
@@ -46,16 +47,17 @@ public function __invoke(
             NameImport::forClass($fqn, $alias);
 
         try {
-            $textEdits = $this->importName->importName(
-                $sourceCode,
-                ByteOffset::fromInt($offset),
-                $nameImport
-            );
+            $byteOffset = ByteOffset::fromInt($offset);
+            if ($updateReferences) {
+                $textEdits = $this->importName->importName($sourceCode, $byteOffset, $nameImport);
+            } else {
+                $textEdits = $this->importName->importNameOnly($sourceCode, $byteOffset, $nameImport);
+            }
```

Comment:
hate to be pedantic: but can we remove the `else` and have one assignment?  `$textEdits = $this->importNameTextEdits($a, $b, $c)`?

Reply:
Done